controllers/common/basicContent.test.js

```diff
@@ -0,0 +1,103 @@
+/* eslint-env jest */
+'use strict';
+const { getChildrenLayoutMode } = require('./index');
+
+describe('CMS page templates', () => {
+    it('should work out feasible child page layout mode', () => {
```

Comment:
❤️ 

btw, if there is only one test block you can skip the `describe` and call `it` (or `test` as simpler alias) on its own